src/app-layout/__tests__/split-panel.test.tsx

```diff
@@ -174,6 +176,57 @@ describeEachAppLayout({ sizes: ['desktop'] }, ({ theme }) => {
     rerender(<AppLayout />);
     expect(wrapper.findSplitPanel()).toBeFalsy();
   });
+
+  test('should fire split panel toggle event', () => {
```

Comment:
Not really new tests, I just moved them up from "classic only" section to this `describeForEachAppLayout` loop